VAMobile/src/screens/HealthScreen/SecureMessaging/CancelConfirmations/ComposeCancelConfirmation.test.tsx

```diff
@@ -1,10 +1,8 @@
 import 'react-native'
 import React from 'react'
-// Note: test renderer must be required after react-native.
-import 'jest-styled-components'
-import { ReactTestInstance, act } from 'react-test-renderer'
 
-import { context, mockNavProps, render, RenderAPI } from 'testUtils'
+import { act } from 'react-test-renderer'
```

Comment:
I don't think we need to use `act`in any of these tests

Reply:
this is needed, fireEvent.press wasn't working cause it used useDestructiveActionSheet hook

Reply:
I was curious, so I tried removing them locally, and the tests still passed for me. Can you confirm on your end also?

Reply:
removed